share/wake/lib/system/job.wake

```diff
@@ -345,6 +357,8 @@ export def mkJobCacheRunner ((Runner name score baseDoIt): Runner): Runner =
 
             Fail e
         Pass (RunnerInput label cmd vis env dir stdin _ prefix _ _) =
+            require Pass client_cwd = pwd Unit
```

Comment:
Why do we need `pwd`? Can we just use `workspace` and/or `here`?

Reply:
I had forgotten about `workspace` at the time, we can remove it for sure